services/t-web/twisted-web/twisted-web

```diff
@@ -247,19 +246,27 @@ httpsService = StreamServerEndpointService(
 httpsService.setServiceParent(application)
 
 if websiteRoot.child('production').exists():
-    issuingService = AcmeIssuingService(
-        cert_store=DirectoryStore(certificates),
-        client_creator=(lambda: Client.from_url(
-            reactor=reactor,
-            url=LETSENCRYPT_DIRECTORY,
-            key=load_or_create_client_key(certificates),
-            alg=RS256,
-        )),
-        clock=reactor,
-        responders=[responder],
+
+    d = Client.from_url(
+        reactor=reactor,
+        url=URL.fromText(
+            u'https://acme-v02.api.letsencrypt.org/directory'),
+        key=load_or_create_client_key(certificates),
+        alg=RS256,
     )
 
-    issuingService.setServiceParent(application)
+    def cb_got_client(client):
+        issuingService = AcmeIssuingService(
+            cert_store=DirectoryStore(certificates),
+            client=client,
```

Comment:
@glyph this is using txacme-v2 form my fork

this is the main change that is breaking the txacme API with v2.

I needed to initiate the client outside of the service, so that for example if the URL is not valid, the service is not started.

At the same time, having control over the client makes testing much easier.

Reply:
Can we just drop txacme entirely and use certbot in webroot mode?  That's effectively what we've been doing manually in production lately.

Reply:
well... txacme-v2 works and is already setup with braid.

I am not -1 on using certbot, but I don't have much time and I would spend the remaining time doing the migration, rather than refactoring braid.

but after the migration we can refactor the code to use certbot.

but the long term plan is to stop using t-web   and just use GitHub Pages + GitHub Wiki + Read The Docs for our websites ... and hopefully we can also migrate to GitHub Issues

Reply:
Sure, if we don't need to mess around with the configuration, it's fine to use this as-is.